connecting-to-testnet/index.md

```diff
@@ -0,0 +1,34 @@
+---
+parent:
+  title: Connecting to a Testnet
+order: 0
+---
+
+# Cosmos Hub Testnet
+
+The [Cosmos Network](https://cosmos.network) is an expanding ecosystem of connected blockchains. Many blockchains in the ecosystem are built with [Tendermint Core](https://docs.tendermint.com/), [Cosmos SDK](https://docs.cosmos.network/) and connected with [IBC](https://ibcprotocol.org/). One of these blockchains is the [Cosmos Hub](https://cosmos.network/features). Mainnet of the Cosmos Hub has been online since early 2019 and you can learn how to [join the network as a validator](https://hub.cosmos.network). In this guide you'll learn how to start a full-node for and interface with a Cosmos Hub **testnet**.
```

Comment:
Let's explain what a testnet is and that state can get reset at any time

Reply:
I think the link for `join the network as a validator` should guide to https://hub.cosmos.network/main/validators/validator-setup.html